### ISSUE TITLE
Unable to remove lambda execute permission to SNS event source

### ISSUE THREAD

**Issue by TammoHeeren** [Miserlou/Zappa#681]
I am getting this recently. At least since 0.36.0. 
Is there something I need to do to fix that?

Tammo

```
Scheduling..
ERROR:Unable to remove lambda execute permission to SNS event source
Traceback (most recent call last):
  File "/home/theeren/.virtualenvs/em30_lambdas/local/lib/python2.7/site-packages/kappa/event_source/sns.py", line 94, in remove
    StatementId=function.name+'_'+self._context.environment)
AttributeError: 'PseudoContext' object has no attribute 'environment'
Removed event app.lambda_handler.
ERROR:Error adding lambdaInvoke permission to SNS event source
Traceback (most recent call last):
  File "/home/theeren/.virtualenvs/em30_lambdas/local/lib/python2.7/site-packages/kappa/event_source/sns.py", line 61, in add
    StatementId=function.name+'_'+self._context.environment,
AttributeError: 'PseudoContext' object has no attribute 'environment'
Created sns:Publish event schedule for app.lambda_handler!
```

**Comment by Miserlou** [comment]
What is your `kappa` version? We updated to 0.7.0, maybe your version in the venv isn't up to date?

**Comment by Miserlou** [comment]
you may also want to try 36.1

**Comment by TammoHeeren** [comment]
It happens with both 36.0 and 36.1. Kappa may not have updated to 0.7.0. I did try to update kappa but it said it was already on the latest version. Maybe it's to recent.

**Comment by TammoHeeren** [comment]
This seems to have got fixed with properly upgrading to kappa 0.7.0.
